Sources/Tracking/CustomerIO.swift

```diff
@@ -251,7 +248,6 @@ public class CustomerIO: CustomerIOInstance {
     }
 
     private func postInitialize(siteId: String) {
-        let diGraph = DIGraph.getInstance(siteId: siteId)
         let threadUtil = diGraph.threadUtil
```

Comment:
similar `diGraph` var removed so whats this refer to now

Reply:
Swift doesn't require you to include `self.` for properties. It automatically determines the scope for you. So, this line of code is referring to `self.`.